src/partials/connectors/connectors-table.hbs

```diff
@@ -19,7 +19,7 @@
     <button class="button js-connector-menu-trigger">
       <svg class="svg" width="16" height="16"><use xlink:href="#icon-kabob"/></svg>
     </button>
-    <div class="js-connector-menu" class="hidden">
+    <div class="js-connector-menu hidden">
```

Comment:
This is missing a closing double quote.

Reply:
It is?  class = open-quote text closed-quote

Reply:
Sorry about that. I misread it. I though you were adding the hidden attribute, but you are adding the hidden class. Both ways work, but this is fine as is.